api/service/staking/service.go

```diff
@@ -87,7 +87,7 @@ func (s *Service) Run() {
 				if s.IsStaked() {
 					return
 				}
-				s.DoService()
+				//s.DoService()
```

Comment:
No need to DoService?

Reply:
Temporarily disable repeated DoService. It's done only once now.